server/src/test/java/org/spine3/server/command/ExecutorCommandSchedulerShould.java

```diff
@@ -69,36 +72,29 @@ public void tearDownTest() {
     }
 
     @Test
-    public void schedule_command_if_delay_is_set() throws InterruptedException {
+    public void schedule_command_if_delay_is_set() {
         final CommandContext context = createCommandContext(DELAY);
         final Command cmd = Commands.create(createProject(newUuid()), context);
 
         scheduler.schedule(cmd);
 
-        assertCommandSent(cmd, DELAY);
+        assertNull(postedCommand);
+        sleep(DELAY_MS + CHECK_OFFSET_MS);
```

Comment:
Please test it using `spy()` or `mock()`. Sleeping threads does not scale well for testing.